extensions/shared/external-media/sources/google-photos/filter-request.js

```diff
@@ -14,9 +32,35 @@ export default function getFilterRequest( filters ) {
 		query.push( 'feature=favorite' );
 	}
 
-	if ( startDate || endDate ) {
-		const start = startDate ? startDate.substr( 0, 10 ) : '0000-00-00';
-		const end = endDate ? endDate.substr( 0, 10 ) : '0000-00-00';
+	if ( date ) {
+		let startDate = null;
+		let endDate = null;
+		switch ( date.range ) {
+			case DATE_RANGE_LAST_7_DAYS:
+				startDate = moment( TODAY ).subtract( 7, 'days' );
```

Comment:
What's the benefit of passing `TODAY` into `moment()`? Does it save us from recreating that object every time?

Reply:
It will actually make a new moment instance. Otherwise, calling the method directly on TODAY will mutate it and will throw all other calculations off